Survey/Survey/Sources/Data/Repositories/LoginRepository.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  LoginRepository.swift
+//  Survey
+//
+//  Created by Taher on 4/11/22.
+//
+
+import Combine
+import Foundation
+
+final class LoginRepository: LoginRepositoryProtocol {
+
+    let network: LoginNetworkAPI
+
+    init(network: LoginNetworkAPI) {
+        self.network = network
+    }
+
+    func getLoginToken(email: String, password: String) -> AnyPublisher<LoginTokenApi, Error> {
```

Comment:
You forgot to use your typealias. Let's update them ^^ 
```suggestion
    func getLoginToken(email: String, password: String) -> Observable<LoginTokenApi> {
```

Reply:
updated in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/43/commits/b1c91288854f7d2e80af7a803aa698093f5ab1f6)